### ISSUE TITLE
Authorization failed

### ISSUE THREAD

**Issue by majkaz** [owenschoppe/Citable#6]
After install, immediately after clicking the "Sign in with Google" I am getting "Authentication failed".

Any idea what might be the reason? Corporate proxy?

There are some mentions of the same problems at the Chrome store, but no solution.

**Comment by owenschoppe** [comment]
Sorry to hear it’s not working. Let me see if I can reproduce the issue. I
may also ask you to check some logs for me, if that’s OK.

Owen

**Comment by owenschoppe** [comment]
Just to confirm, are you using Google Chrome? Latest version?

I just did a fresh install and I was able to sign in without issue. If you are able, please do the following to help me debug the issue: from the extension management page, click inspect background, try authenticating again, and then from the background console copy the log and share it here.

**Comment by majkaz** [comment]
OK, the mystery seems to be solved:
I have disabled browser sign in - the log follows

```
gdocs accessToken undefined

background.html:1 Unchecked runtime.lastError: The user turned off browser signin
```

It might be worth mentioning on the extension page, that browser sign in is needed. 

I had to fight the sign in even on my home PC, where browser sign in is switched on - the login seemed to work, but was cycling several times - it didn't look like the sign in was sticking. I got it to work but I have no idea how - there was nothing different from the previous tries.
